sources/packages/api/src/route-controllers/assessment/models/assessment.dto.ts

```diff
@@ -21,14 +21,16 @@ export class RequestAssessmentSummaryAPIOutDTO {
 }
 
 export class AssessmentHistorySummaryAPIOutDTO {
-  assessmentId: number;
+  assessmentId?: number;
   submittedDate: Date;
   triggerType: AssessmentTriggerType;
-  assessmentDate: Date;
+  assessmentDate?: Date;
   status: StudentAssessmentStatus;
   studentAppealId?: number;
   applicationExceptionId?: number;
   studentScholasticStandingId?: number;
+  // This flag decides, the row is unsuccessful week or not.
+  unsuccessfulWeek?: boolean;
```

Comment:
I would rename to hasUnsuccessfulWeeks.

Reply:
i will add `isUnsuccessfulWeek`

Reply:
I would agree with `hasUnsuccessfulWeeks`.